cpp/oneapi/dal/detail/memory_impl_dpc.hpp

```diff
@@ -33,6 +33,14 @@ ONEDAL_EXPORT void memcpy(const data_parallel_policy& policy,
                           void* dest,
                           const void* src,
                           std::int64_t size);
+ONEDAL_EXPORT void memcpy_usm2host(const data_parallel_policy& policy,
+                                   void* dest_host,
+                                   const void* src_usm,
+                                   std::int64_t size);
+ONEDAL_EXPORT void memcpy_host2usm(const data_parallel_policy& policy,
+                                   void* dest_usm,
+                                   const void* src_host,
+                                   std::int64_t size);
```

Comment:
I do not see usage of those functions. Do we have `memcpy` in the header files that should be replaced by one those functions?

Reply:
These functions are exposed in public API to use it internally in benchmarks